server/mldb_server.cc

```diff
@@ -263,6 +271,32 @@ runHttpQuery(const Utf8String& query,
                        rowNames, rowHashes, sortColumns);
 }
 
+void
+MldbServer::
+handleRedirectToGet(const string & uri,
+                    RestConnection & connection,
+                    const Json::Value & body) const
+{
```

Comment:
Is there any risk that someone could make a call to this route to get something not related to mldb?
Say I am on a LAN. I have no access to the extranet but I have access to MLDB. MLDB has access to the extranet. Can I use MLDB to get out?

This is a security risk question.